pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
:+1: @PI-Victor 

The intention of this name is to tell what "post build action" it is. In a previous iteration we had "StartBuild", we need something in that direction, as in the future there might be other fields/actions within `BuildPostCommitSpec`.

We also need to document clearly what this string means without requiring you to go read the docs every time you want to use it. Reiterating what @PI-Victor said, we chose bash. That's what other CI tools offer, as we gathered in our R&D.

Suggestions that clearly state that this is one action out of other possible actions (even if they don't exist yet) and that this is a Bash script are welcome.